src/backend/gog/games.ts

```diff
@@ -421,14 +466,24 @@ class GOGGame extends Game {
       `Launch Command: ${fullCommand}\n\nGame Log:\n`
     )
 
-    const { error } = await runGogdlCommand(commandParts, {
-      env: commandEnv,
-      wrappers,
-      logMessagePrefix: `Launching ${gameInfo.title}`,
-      onOutput: (output: string) => {
-        appendFileSync(this.logFileLocation, output)
+    const { error, abort } = await runGogdlCommand(
+      commandParts,
+      createAbortController(this.appName),
+      {
+        env: commandEnv,
+        wrappers,
+        logMessagePrefix: `Launching ${gameInfo.title}`,
+        onOutput: (output: string) => {
+          appendFileSync(this.logFileLocation, output)
+        }
       }
-    })
+    )
+
+    deleteAbortController(this.appName)
+
+    if (abort) {
+      return true
```

Comment:
Depends, as I said above about the downloads. Heroic launched the game just fine, if the user clicked on STOP for some reason, it was aborted due to user interaction/choice, Heroic launched the game just fine so I think yes.

Reply:
We want to log a abort as abort and not as error. The problem was that we never handled abort correctly. We always showed alot of error messages, like game failed to install and alot more. We don't want to log that because aborts via abortcontroller are intenional.